pkg/controller/bypass/bypass_controller.go

```diff
@@ -124,6 +146,46 @@ func StartByPassController(client kubernetes.Interface) error {
 	return nil
 }
 
+func handleKmeshBypass(ns string, port int) error {
+	execFunc := func(netns.NetNS) error {
+		/*
+		 * Attempt to connect to a special IP address. The
+		 * connection triggers the cgroup/connect4 ebpf
+		 * program and records the netns cookie information
+		 * of the current connection. The cookie can be used
+		 * to determine whether the netns is been bypass.
+		 * 0.0.0.1:<port> is "cipher key" for cgroup/connect4
+		 * ebpf program.
+		 */
```

Comment:
The meanings of the special ports(931/932) need to described in the comments.

Reply:
> The meanings of the special ports(931/932) need to described in the comments.

Added